src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person,
+  mother: Person | undefined,
+  father: Person | undefined,
```

Comment:
```suggestion
  mother?: Person,
  father?: Person,
```

should work same but simplify